x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityFcActionAuthorizationIT.java

```diff
@@ -282,7 +282,7 @@ public void testIndicesPrivilegesAreEnforcedForCcrRestoreSessionActions() throws
                 GetCcrRestoreFileChunkAction.REMOTE_TYPE,
                 new GetCcrRestoreFileChunkRequest(response2.getNode(), sessionUUID2, leaderIndex2FileName, 1, shardId2)
             );
-            assertFalse(getChunkResponse.getChunk().hasReferences());
+            assertBusy(() -> assertFalse(getChunkResponse.getChunk().hasReferences()));
```

Comment:
Backport of: https://github.com/elastic/elasticsearch/pull/127888

After unmuting `testIndicesPrivilegesAreEnforcedForCcrRestoreSessionActions`, it was still failing.